src/openMVG/matching/indMatch.hpp

```diff
@@ -85,4 +107,9 @@ inline Pair_Set getPairs(const PairWiseMatches & matches)
 }  // namespace matching
 }  // namespace openMVG
 
+namespace cereal
+{
+    // This struct specialization will tell cereal which is the right way to serialize PairWiseMatches
```

Comment:
Does void serialize( Archive & ar )  { is still used since you have added this new specialization?